pages/_app.js

```diff
@@ -21,6 +22,13 @@ export default function App({ Component, pageProps }) {
     setFormResults(newResults);
   }
 
+  function handleSurprise() {
+    const randomPlace = places[Math.floor(Math.random() * places.length)];
+    setRandomSurprise(randomPlace);
+    console.log(randomSurprise);
+    return randomSurprise;
```

Comment:
you don't need to return anything here.

You set the state within the `handleSurprise` function and that is all this functions needs to do.